app/helpers/email_signup_helper.rb

```diff
@@ -0,0 +1,24 @@
+module EmailSignupHelper
+
+  def different_email_signup_path(atom_url)
```

Comment:
I was thinking different as in substituting it but I guess it's only different if it matches the regex so I'll change it.